### ISSUE TITLE
Roboto mono seems to be missing some box drawing characters

### ISSUE THREAD

**Issue by kjpgit** [google/fonts#6653]
Using chrome secure shell with roboto mono font.

U+2500 seems to be missing, so it fallsback to another font which has a different width, so my terminal lines are not aligned  :(

These chars have been around for probably 50 years, and we still can't get them working :(

**Comment by eliheuer** [comment]
Yes, Roboto Mono should probably have the main box drawing unicode block characters. Roboto Mono is open source software, you or anyone else if free to add them in a fork an make a pull request upstream here: https://github.com/googlefonts/RobotoMono

I'm leaving this open because this this should happen eventually.
